articles/most-useful-node.js-packages/index.md

```diff
@@ -0,0 +1,349 @@
+## Most Useful Node.js Packages That you need to Know
+Open-source frameworks that forms the ecosystem of JavaScript back the power of Node.js. As a Node.js developer, it is practically impossible to constantly avoid their advantage. With the help of Node.js packages, as a developer, you will conclusively save a lot of time and develop applications that are light, scalable, high speed with tremendous productivity.
+Let's have a look at the packages that will assist you to extend your inclination to Node.js application development.
+### 1. Express
+Express.js is the fastest, unopinionated, and simplest web framework for Node.js. It is very flexible designed for building simple page, multipage and hybrid applications that provide a robust set of features for web and mobile applications. The framework is minimal. It is the leading Node.js framework designed for building web applications and
```

Comment:
Should be flexibly not flexible. 

Should be Sails and NestJS. Not all caps. They should be seperated by and not a comma.